starknet_py/net/client_models.py

```diff
@@ -32,6 +32,15 @@ class Event:
     data: List[int]
 
 
+@dataclass
+class Events:
```

Comment:
Do we have to create a dataclass for a list of Events? I think `get_events` could just return `List[Event]`.
cc @drknzz @cptartur

Reply:
There's also `continuation_token` which was not included.